admission-webhook/deploy/deploy-gmsa-webhook.sh

```diff
@@ -150,15 +146,21 @@ main() {
         effect: NoSchedule'
     fi
 
-    TLS_PRIVATE_KEY=$(cat "$SERVER_KEY" | base64 -w 0) \
+    if [ -x "$(command -v envsubst)" ]; then
+        ENVSUBST="$(command -v envsubst) < $TEMPLATE_PATH"
+    else
+        ENVSUBST="docker run --rm -v $TEMPLATE_PATH:$TEMPLATE_PATH --env-file <(env) supinf/envsubst $TEMPLATE_PATH"
```

Comment:
I think the -v flag should be `-v $(dirname "$TEMPLATE_PATH"):/tmp` instead?

Reply:
-v also supports mounting single files, so for simplicity I opted to just mount the template file at the same path as on the host:

ubuntu@ubuntu18:~$ cat test.txt
just a test
ubuntu@ubuntu18:~$ sudo docker run --rm -v /home/ubuntu/test.txt:/home/ubuntu/test.txt ubuntu:latest cat /home/ubuntu/test.txt
just a test

Reply:
TIL :) thank you!